src/components/DashboardPage/index.js

```diff
@@ -131,7 +129,7 @@ class DashboardPage extends PureComponent<Props, State> {
         <Box horizontal alignItems="flex-end">
           <Box grow>
             <Text color="dark" ff="Museo Sans" fontSize={7}>
-              {t('dashboard:greetings', { name: username })}
+              {t('dashboard:greetings')}
```

Comment:
does the greedings make sense at all now 🤔  ?

Reply:
I asked Brian what should be there and he said 'Good morning'

Reply:
What if it's the night ? Do we plan to update the greeting according to time of the day ? What about displaying meteo infos ? :trollface:

Reply:
We could say "Greetings!" to still have a greeting or if we have to have morning/evening one we should probably have a story to make a greeting dynamic